bot/admin/admin.py

```diff
@@ -316,6 +414,50 @@ def _create_cogs_embed_string(loaded_cogs: Mapping[str, commands.Cog]) -> str:
     return string
 
 
+def _build_botonly_embed(is_enabled_string: str):
+    """Creates an embed for the botonly command.
+
+    Args:
+        is_enabled_string (str): A string which will be interpolated into the title. Should contain the word 'aktiviert'
+        or 'deaktiviert'.
+
+    Returns:
+        (discord.Embed): An embed containing information, if the botonly feature was en- or disabled for a channel.
+    """
+    title = 'Bot-Only Funktionalität wurde für diesen Channel {0}'.format(is_enabled_string)
+    description = 'Ein Bot-Only Channel ist ein Channel in dem nur ein Bot Nachrichten posten kann. Jede Nachricht von' \
+                  'anderen Usern wird sofort gelöscht.'
+    return discord.Embed(title=title, description=description, color=constants.EMBED_COLOR_BOTONLY)
+
+
+def _build_purge_confirmation_embed(channel: discord.TextChannel) -> discord.Embed:
+    """Creates an embed for confirmation of the purge command.
+
+    Args:
+        channel (discord.TextChannel): The channel that will be mentioned in the embed message.
+
+    Returns:
+        (discord.Embed): The embed with the confirmation dialog
+    """
+    title = 'Bist du sicher dass du den Channel {0} purgen möchtest?'.format(channel.name)
+    description = 'Wenn du den Purge Befehl ausführst, werden sämtliche Nachrichten in dem Channel gelöscht. ' + \
+                  'Diese Operation kann man nicht rückgängig machen. Bitte beachte dass das löschen aller Nachrichten ' + \
+                  'ein wenig dauern kann.'
+    return discord.Embed(title=title, description=description, color=constants.EMBED_COLOR_WARNING)
+
+
+async def _purge_channel(channel: discord.TextChannel):
```

Comment:
KILL IT WITH FIRE! 🔥
But as said before: A `purge` command should only exist with an argument for the amount of messages to be removed and a hard coded limit.

Reply:
purge command has been commented out, will be handled in separate issue.